app/modules/site_settings/models.py

```diff
@@ -111,6 +111,11 @@ def __repr__(self):
     def is_public(self):
         return self.public
 
+    @classmethod
```

Comment:
The default query_search has been overwritten here. Can you add a comment to say why this was necessary in this module?

Reply:
The `def query_search()` class method is defined on the `FeatherModel` and is inherited by all other models in Houston.  The `SiteSetting` model is defined as `class SiteSetting(db.Model, Timestamp)` and does not inherit from `FeatherModel` or `HoustonModel`.  SiteSettings do not inherit from Feather or Houston Models because we need them very early on in the startup's import bootstrapping, before the `app.extension` module has even been loaded.  This means that some of the features need to be copied into this class to add functionality for Elasticsearch, which supports the `SiteSetting` model.

Reply:
OK, Could you add a comment to that effect in the code as it jumps out as being odd when you look at it and I can see time being wasted if people thinnk SiteSetting should really inherit from HoustoModel.

Reply:
Added.